### PR TITLE
[Community Pipelines] K-Diffusion Pipeline

### PR DESCRIPTION
**Pull request by patrickvonplaten**
This PR adds a pipeline that supports k-diffusion

### CODE REVIEW

File: examples/community/README.md [hunk 1]
```diff
@@ -663,4 +664,65 @@ Based https://arxiv.org/abs/2208.04202, this is used for diffusion on discrete d
...
+pipe.scheduler = EulerDiscreteScheduler.from_config(pipe.scheduler.config)
+pipe = pipe.to("cuda")
+
+pipe.set_sampler("sample_euler")
+generator = torch.Generator(device="cuda").manual_seed(seed)
+image = pipe(prompt, generator=generator, num_inference_steps=50).images[0]
+```
+
+![diffusers_euler](https://huggingface.co/datasets/patrickvonplaten/images/resolve/main/k_diffusion/astronaut_euler_k_diffusion.png)
+
```
Comment (pingren) [inline examples/community/README.md:728]: Hi @patrickvonplaten 

I can't tell the visual difference of these two images:

diffusers_euler

diffusers_euler with k-diffusion

I suppose there would be something different to compare?

Comment (pingren) [inline examples/community/README.md:728]: Hi @patrickvonplaten 

I can't tell the visual difference of these two images:

diffusers_euler

diffusers_euler with k-diffusion

I suppose there would be something different to compare?

Comment (patrickvonplaten) [reply examples/community/README.md:728]: Hey @pingren, 

All I wanted to show here is that it "works" :sweat_smile:

Comment (pingren) [reply examples/community/README.md:728]: I saw it in v0.8 a release notes and want to see what's the new k-diffusion, and it confuses me by showing “almost identical” results of K-diffusion and original diffusion.

Do you like the idea to tweak the parameters and show the difference? Reduce steps from 50 to 20 will bring visual differences.

CleanShot 2022-11-30 at 12 02 11

**Pull request merged**